### PR TITLE
Refactor CSS files in src/screens/OrgList

### LINKED ISSUE #2521: Refactor CSS files in src/screens/OrgList
# Refactor CSS files in src/screens/OrgList

To improve the UI/UX of Talawa-Admin we have been working on:
1. making the application more suitable to color blind users
2. streamlining all CSS into a single global file

## Additional information:
1. The foundational work for this was completed in this PR:
    - https://github.com/PalisadoesFoundation/talawa-admin/pull/2466
2. The desired single CSS file is:
    - `src/style/app.module.css`

## The challenge:
The goal is to convert the CSS file in this subdirectory and all the components related to this screen to use this new design pattern. This means that:
1. All CSS files in this subdirectory and all the CSS files in related components must be merged into the global CSS file using pre-existing CSS.
2. All CSS files in this subdirectory and all the CSS files in related components must be deleted after the merge.
3. The remaining relevant files in this subdirectory must only reference the global CSS file.
4. All the CSS files in the related components of this subdirectory must adhere to the same guidelines.

### PR DESCRIPTION
**Pull request by abbi4code**
**What kind of change does this PR introduce?**
Streamlined all CSS for OrgList into a single global file, fixed UI bugs on the OrgList page, and ensured no conflicts with other pages.

**Issue Number:**

Fixes #2521 

**Snapshots/Videos:**
**Resolved minor UI and CSS issues on the OrgList page.**

- Before

Screenshot from 2024-12-24 12-29-45

-After

Screenshot from 2024-12-24 14-10-19

## Summary by CodeRabbit

- **New Features**
	- Introduced new button styles and layout classes for the OrgList component, enhancing user interaction and visual organization.
	- Added functionality to trigger the creation of a sample organization directly from the Organization Modal.

- **Bug Fixes**
	- Improved styling consistency across the OrgList interface with updated class names.

- **Documentation**
	- Updated the interface for organization modal properties to include new functionality.

- **Style**
	- Comprehensive updates to styles for the OrgList component, including responsive design elements and new utility classes.

### CODE REVIEW

**Comment by palisadoes** [comment]
Please ensure coderabbit.ai approves your work

**Comment by abbi4code** [comment]
@coderabbitai review

**Comment by abbi4code** [comment]
> Please ensure coderabbit.ai approves your work

Done

File: src/style/app.module.css [hunk 1]
```diff
@@ -1282,3 +1282,342 @@
...
+.searchButtonOrgList {
+  position: absolute;
+  z-index: 10;
+  bottom: 0;
+  inset-inline-end: 0px;
+  height: 100;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```
[the remark this answers is not in the document]
Comment (abbi4code) [reply src/style/app.module.css:1322]: working on it

**Review by palisadoes — APPROVED** [review]

**Pull request merged**